pkg/solana/relay.go

```diff
@@ -187,6 +187,8 @@ func (r *Relayer) NewAutomationProvider(ctx context.Context, rargs relaytypes.Re
 
 var _ relaytypes.ConfigProvider = &configProvider{}
 
+type GetReader func() (client.Reader, error)
+
```

Comment:
why is this defined here?

`GetAccountReader` is defined where it is being used...